library/spdm_responder_lib/libspdm_rsp_measurements.c

```diff
@@ -175,6 +175,12 @@ libspdm_return_t libspdm_get_response_measurements(void *context,
             spdm_context, SPDM_ERROR_CODE_UNSUPPORTED_REQUEST,
             SPDM_GET_MEASUREMENTS, response_size, response);
     }
+    if ((spdm_context->connection_info.algorithm.measurement_spec == 0) ||
+        (spdm_context->connection_info.algorithm.measurement_hash_algo == 0) ) {
```

Comment:
`spdm_context->connection_info.algorithm.measurement_hash_algo` is set to zero if the Responder supports measurements but its `MeasurementHashAlgo` is raw bit stream only (0), correct? In that case it obviously shouldn't return an error.

Reply:
measurement_hash_algo is set to BIT0 (1) if raw bit stream only.

It is BIT0, the value is 1, not 0.

Reply:
Ah yeah that's right. I keep forgetting that's a bit mask.